api_docs/playlists.md

```diff
@@ -81,6 +82,12 @@ playlist[rental_required] | Enable rental for the playlist *edge feature* | Bool
 playlist[rental_price] | Rental price for the playlist bundle, sent as a string, ie "4.99" *edge feature* | String
 playlist[rental_duration] | Rental duration for the playlist, in days *edge feature* | Integer
 playlist[auto_update_consumer_videos] | Keep consumer video purchases and/or rentals in sync with playlist items. When videos are added to the playlist, also add them to a consumer's video purchase/rental list. *edge feature* | Boolean
+playlist[thumbnail_layout] | Update preferred thumbnail layout for videos related to playlist. Options are 'landscape', 'poster' and 'square'. | String
+playlist[images_attributes][][title] | Title of the image | String
```

Comment:
can we add "playlist[images_attributes][][id]" to the DEV docs? we need this parameters to update or delete a playlist image.